documentation/internal/resource-manager-active-directory.md

```diff
@@ -0,0 +1,272 @@
+
+_A JavaScript Object Notation (JSON) file that defines one or more resources to deploy to a resource group or subscription. The template can be used to deploy the resources consistently and repeatedly._
+
+Users can find sample JSON templates to deploy in the [`azure-quickstart-templates`](https://github.com/Azure/azure-quickstart-templates) repository. The deployment cmdlets found in the `Az.Resources` module are used to manage the deployment of these templates.
+
+The cmdlets currently found in the `Az.Resources` module that act upon deployments are the following:
+
+- [`Get-AzDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Get-AzDeployment.md)
+    - Retrieves all of the deployments in the current subscription. The user can also filter the results by name and ID.
+- [`Get-AzResourceGroupDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Get-AzResourceGroupDeployment.md)
+    - Retrieves all of the deployments in the specified resource group. The user can also filter the results by name and ID.
+- [`New-AzDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/New-AzDeployment.md)
+    - Creates a new deployment in the current subscription.
+- [`New-AzResourceGroupDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/New-AzResourceGroupDeployment.md)
+    - Creates a new deployment in the specified resource group.
+- [`Remove-AzDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Remove-AzDeployment.md)
+    - Deletes a deployment in the current subscription with the given name or ID.
+- [`Remove-AzResourceGroupDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Remove-AzResourceGroupDeployment.md)
+    - Deletes a deployment in the specified resource group with the given name or ID.
+- [`Stop-AzDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Stop-AzDeployment.md)
+    - Cancels a deployment in the current subscription with the given name or ID.
+- [`Stop-AzResourceGroupDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Stop-AzResourceGroupDeployment.md)
+    - Cancels a deployment in the specified resource group with the given name or ID.
+- [`Test-AzDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Test-AzDeployment.md)
+    - Determines whether the given deployment template and its parameter values are valid in the current subscription.
+- [`Test-AzResourceGroupDeployment`](https://github.com/Azure/azure-powershell/blob/master/src/Resources/Resources/help/Test-AzResourceGroupDeployment.md)
+    - Determines whether the given deployment template and its parameter values are valid in the specified resource group.
+
+#### Policy Definition
```

Comment:
So, I was thinking this would mostly be about the parts of this that we own, particularly the generic resource cmdlets, and the new graph.rabc cmdlets.  This other stuff about policy and role assignments is also useful, but not the intended focus.

Reply:
@markcowl updated -- stripped out all of the sections that we don't own, included examples for piping scenarios per cmdlet (where applicable) and called out any user scenarios we've implemented.